### PR TITLE
Update alpha API to beta

### PR DESCRIPTION
**Pull request by bswartz**
**What type of PR is this?**
/kind api-change

**What this PR does / why we need it**:
Change alpha API to beta

**Does this PR introduce a user-facing change?**:
```release-note
Renamed v1alpha1 API to v1beta1. The API is the same, but users must use the new version.
```

### CODE REVIEW

**Comment by chrishenzie** [comment]
I didn't expect that to go through without approval, apologies @xing-yang

**Comment by bswartz** [comment]
> I didn't expect that to go through without approval, apologies @xing-yang

Looks like something causes me to auto-approve my own PRs. I wonder if that can be reversed.

**Comment by bswartz** [comment]
@xing-yang Any feedback/concerns you have I can address in a new PR, just let me know.

**Comment by xing-yang** [comment]
@chrishenzie @bswartz no worries.  I'll take a look.  If I have comments, Ben can address with a followup PR.

File: client/config/crd/populator.storage.k8s.io_volumepopulators.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
     singular: volumepopulator
   scope: Namespaced
   versions:
-  - name: v1alpha1
+  - name: v1beta1
```
Comment (xing-yang) [inline client/config/crd/populator.storage.k8s.io_volumepopulators.yaml:20]: Should line 8 be updated? 
```
api-approved.kubernetes.io: https://github.com/kubernetes/enhancements/pull/2056
```

**Pull request merged**